mod_reforged/hooks/config/strings.nut

```diff
@@ -1001,7 +1008,7 @@ foreach (vanillaDesc in vanillaDescriptions)
 		Effects = [{
 			Type = ::UPD.EffectType.Active,
 			Description = [
-				"Unlocks the [Blitzkrieg|Skill+rf_blitzkrieg_skill] skill which allows you and the rest of your company to go first in the next round of combat."
+				"Unlocks the [Blitzkrieg|Skill+rf_blitzkrieg_skill] skill which allows you and the rest of your company to gain the [Adrenaline|Skill+adrenaline_effect] effect."
```

Comment:
I prefer the original wording here.

Reply:
But it's significant that it's adrenaline because adrenaline now ignores injuries.

Reply:
Ah yes that's a good point. Yeah then this looks good.